bikespace_frontend/src/components/dashboard/filter-date-range-custom/FilterDateRangeCustom.tsx

```diff
@@ -84,11 +86,7 @@ export function FilterDateRangeCustom() {
           type="date"
           id="filter-start-date"
           name="startDate"
-          value={
-            selectedDateRange.from
-              ? formatHtmlDateValue(selectedDateRange.from)
-              : isoFirst || ''
-          }
+          value={formatHtmlDateValue(selectedDateRange.from || today)}
           min={isoFirst!}
```

Comment:
I was thinking for this and the other `input=date` element, we can probably remove the min and max values. 

After writing some tests for this component, I've realized that these built-in validation fields aren't testable and often only work in current implementations when the user clicks through the UI instead of typing.

More importantly, the user might want to define a date period that includes a future date, even if that date is later than the most recent report's `parking_time` value. In fact, the "this year" preset does this already.